Marlin/macros.h

```diff
@@ -4,6 +4,8 @@
 // Macros for bit masks
 #define BIT(b) (1<<(b))
 #define TEST(n,b) (((n)&BIT(b))!=0)
+#define BITSET(n,b) n |= BIT(b)
+#define BITCLR(n,b) n &= ~BIT(b)
 #define SET_BIT(n,b,value) (n) ^= ((-value)^(n)) & (BIT(b))
```

Comment:
How about using the already existent and a bit more 'standard' sbi(), cbi()?
https://github.com/MarlinFirmware/Marlin/blob/RC/Marlin/Marlin.h#L48